app/views.py

```diff
@@ -26,7 +26,8 @@ def make_url():
     return base + urllib.parse.urlencode(params)
 
 def home(request):
-    is_authed = bool(request.COOKIES.get('access_token', ''))
+    access_token = request.COOKIES.get('access_token', '')
+    is_authed = bool(access_token)
```

Comment:
I think this could be clearer, what do you expect this value to be if they're not authed, vs if they are, and put in a specific check for that?
Like `is_authed = access_token != None` or something like that